src/partition/repart.c

```diff
@@ -4775,6 +4775,18 @@ static int make_subvolumes_set(
         return 0;
 }
 
+static usec_t epoch_or_utime_omit(void) {
+        uint64_t epoch;
```

Comment:
so, this function is something that might be called a lot, and it returns the same thing each time. Hence, I think it should really have a cache, i.e. just add:

```c
static uint64_t cache = UINT64_MAX;

if (cache != UINT64_MAX)
        return cache;

…
```

and then initialize `cache` to what you return. you get the idea.